internal/guest/bridge/bridge_v2.go

```diff
@@ -423,10 +416,8 @@ func (b *Bridge) resizeConsoleV2(r *Request) (_ RequestResponse, err error) {
 }
 
 func (b *Bridge) modifySettingsV2(r *Request) (_ RequestResponse, err error) {
-	ctx, span := oc.StartSpan(r.Context, "opengcs::bridge::modifySettingsV2")
-	defer span.End()
-	defer func() { oc.SetSpanStatus(span, err) }()
-	span.AddAttributes(trace.StringAttribute("cid", r.ContainerID))
+	ctx := r.Context
+	log.G(ctx).Trace("opengcs::bridge::modifySettingsV2")
```

Comment:
Same question here. Should this and `dumpStacksV2` use `logEntry`?

Reply:
`dumpStacksV2` doesnt apply to a particular container, right?
but yes, `deleteContainerStateV2` should